requirements.txt

```diff
@@ -188,4 +189,4 @@ websocket-client==1.8.0
 werkzeug==3.0.3
 widgetsnbextension==4.0.10
 wrapt==1.14.1
-zipp==3.18.1
+zipp==3.18.1
```

Comment:
Did anything change here?

Reply:
It did not, might be a typo it github picked up